src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ProgressBar } from '../ProgressBar';
+import { OverlayTrigger } from '../OverlayTrigger';
+import { Tooltip } from '../Tooltip';
+import { labelClasses, mainDivClasses, barStyle } from './helpers';
+
+const randomId = () => Date.now();
+
+const AvailableTooltipFunction = (max, now) => (
+  <Tooltip id={randomId()}>Available {max - now} %</Tooltip>
+);
+
+const UsedTooltipFunction = now => (
```

Comment:
We should pass `max` and `now` here as well for overrides that need to know both.